Aga8Tests/Aga8CalcTests.cs

```diff
@@ -109,5 +132,55 @@ public async Task Aga8OpcClient_Constructor()
             Assert.IsTrue(client.OpcSession.Connected);
             client.DisConnect();
         }
+
+        [TestClass]
+        public class ConfigTests
+        {
+            const double accuracy= 0.000_01;
+
+            [TestMethod]
+            public void GetConvertedTemperature_UnitIsCelsius_ReturnKelvin()
+            {
+                var config = new Config();
+                double testTempCelsius = 446.3;
+                config.Temperature = 173.15;
+
+                double resultCelsius = config.GetConvertedTemperature(Config.TemperatureUnits.C);
```

Comment:
This should be named resultKelvin, since GetConvertedTemperature always returns the temperature in Kelvin.